src/Controls/src/Core/VisualElement/VisualElement.cs

```diff
@@ -702,7 +702,7 @@ private protected bool InputTransparentCore
 		/// All types that override this property need to also invoke
 		/// the RefreshIsVisibleProperty() method if the value will change.
 		/// </summary>
-		private protected bool IsVisibleCore
+		public bool IsVisibleCore
```

Comment:
This should be protected.

```suggestion
		protected bool IsVisibleCore
```